auditlog/models/rule.py

```diff
@@ -343,6 +358,16 @@ def read(self, fields=None, load="_classic_read", **kwargs):
                 return result
             self = self.with_context(auditlog_disabled=True)
             rule_model = self.env["auditlog.rule"]
+            rule = rule_model.sudo().search([("model_id.model", "=", self._name)])
+            if self.env.user in rule.mapped("users_to_exclude"):
+                return result
+            fields_to_exclude_match = [
+                d in rule.fields_to_exclude.mapped("name") for d in fields
+            ]
+            if (len(fields) == 1 and any(fields_to_exclude_match)) or (
+                all(fields_to_exclude_match)
+            ):
+                return result
```

Comment:
What makes 1 special? `any([x]) == all([x])` anyway

Reply:
@coleste we optimized the code and this line section is removed now